std/assembly/util/string.ts

```diff
@@ -49,13 +49,27 @@ const Powers10Lo: f64[] = [
 ];
 
 export function compareImpl(str1: string, index1: usize, str2: string, index2: usize, len: usize): i32 {
-  var result = 0;
   var ptr1 = changetype<usize>(str1) + (index1 << 1);
   var ptr2 = changetype<usize>(str2) + (index2 << 1);
-  while (len && !(result = <i32>load<u16>(ptr1) - <i32>load<u16>(ptr2))) {
-    --len, ptr1 += 2, ptr2 += 2;
+  if (ASC_SHRINK_LEVEL < 2) {
+    if (len == 1) return <i32>load<u16>(ptr1) - <i32>load<u16>(ptr2);
```

Comment:
Not sure if worth it

Reply:
Yeah. Thinking more about this I guess this pretty rare case. I'll remove this